### PR TITLE
Ava UI: Fix double click anywhere

### PR DESCRIPTION
**Pull request by IsaacMarovitz**
This fixes an issue that allowed you to double-click anywhere within the list box and open the selected game. Now it will only open if the game's list box item was what was clicked. I'm not a big fan of this solution, but putting the double-click handler on the list box item itself resulted in some inconsistency in where you could click and have it register.

### CODE REVIEW

File: src/Ryujinx.Ava/UI/Controls/ApplicationGridView.axaml.cs [hunk 1]
```diff
@@ -42,10 +45,5 @@ public void GameList_SelectionChanged(object sender, SelectionChangedEventArgs a
                 (DataContext as MainWindowViewModel).GridSelectedApplication = listBox.SelectedItem as ApplicationData;
             }
         }
-
-        private void SearchBox_OnKeyUp(object sender, KeyEventArgs args)
-        {
-            (DataContext as MainWindowViewModel).SearchText = (sender as TextBox).Text;
-        }
```
Comment (marysaka) [inline src/Ryujinx.Ava/UI/Controls/ApplicationGridView.axaml.cs:49]: Why was that removed? Is it unused?

File: src/Ryujinx.Ava/UI/Controls/ApplicationListView.axaml.cs [hunk 2]
```diff
@@ -42,10 +46,5 @@ public void GameList_SelectionChanged(object sender, SelectionChangedEventArgs a
                 (DataContext as MainWindowViewModel).ListSelectedApplication = listBox.SelectedItem as ApplicationData;
             }
         }
-
-        private void SearchBox_OnKeyUp(object sender, KeyEventArgs args)
-        {
-            (DataContext as MainWindowViewModel).SearchText = (sender as TextBox).Text;
-        }
```
Comment (marysaka) [inline src/Ryujinx.Ava/UI/Controls/ApplicationListView.axaml.cs:49]: Why was that removed? Is it unused?

**Review by marysaka — APPROVED** [review]
Not really a fan of your solution but oh well...

**Comment by AcK77** [comment]
Same comments as @marysaka here. Once you explain, I'll approve ;)
